cpp/ql/src/Security/CWE/CWE-078/ExecTainted.ql

```diff
@@ -126,14 +134,7 @@ class ExecTaintConfiguration extends TaintTracking::Configuration {
     state2.(ExecState).getOutgoingNode() = node2
   }
 
-  override predicate isSanitizer(DataFlow::Node node, DataFlow::FlowState state) {
-    (
-      node.asInstruction().getResultType() instanceof IntegralType
-      or
-      node.asInstruction().getResultType() instanceof FloatingPointType
-    ) and
-    state instanceof ConcatState
```

Comment:
The restriction on the state seemed to have been removed, was that intentional?

Reply:
Oh yes, good point. I actually couldn't figure out why we only wanted the the sanitizer to kick in in the `ConcatState` so I just removed that restriction.

Reply:
I suspect that might have been a left-over from when these were two separate configs stitched together, and this might only have occurred in the config which is now `ConcatState`. So it might indeed be unnecessary.

Reply:
Looking at [where it was introduced](https://github.com/github/codeql/commit/f8198c312389b5b6c78d040462fcbc15e6a53d80) I guess the restriction on the state was put there only because the sanitizer was only present in one of the original configurations, but I think it should've been in both?

Reply:
I think we arrived at the same conclusion concurrently 😂. I agree completely!

Reply:
Yeah.